client/my-sites/themes/helpers.ts

```diff
@@ -1,18 +1,34 @@
 import { mapValues } from 'lodash';
+import { BaseSyntheticEvent } from 'react';
 import titlecase from 'to-title-case';
 import { gaRecordEvent } from 'calypso/lib/analytics/ga';
 import { isMagnificentLocale } from 'calypso/lib/i18n-utils';
 
-export function trackClick( componentName, eventName, verb = 'click' ) {
+interface ThemeOption {
+	action?: () => void;
```

Comment:
Unless there's a linter rule forbidding that, I'd prefer using shorthand here and on the other properties as well:

```suggestion
	action?(): void;
```

It's just shorter 😄